src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Persistence/Services/AssignmentService.cs

```diff
@@ -29,6 +30,136 @@ IDelegationRepository delegationRepository
     private readonly IRolePackageRepository rolePackageRepository = rolePackageRepository;
     private readonly IEntityRepository entityRepository = entityRepository;
     private readonly IDelegationRepository delegationRepository = delegationRepository;
+    private static readonly string RETTIGHETSHAVER = "rettighetshaver";
+    private static readonly Guid PartyTypeOrganizationUuid = new Guid("8c216e2f-afdd-4234-9ba2-691c727bb33d");
```

Comment:
litt usikker på hvordan vi bør stykke opp disse statiske UUIDene / rollene. Vi passer det rundt som string literals eller definerer en lokal statisk verdi. Topikk for anotha day, har gjort det selv flere steder, men virker litt "magisk".

Reply:
Veldig enig!